{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/home/urls.py

```diff
@@ -7,5 +7,3 @@
     url(r'^error/', error, name='error'),
     url(r'^$', TemplateView.as_view(template_name="index.html"), name="home"),
 ]
-
-from .signals import * #ensure that the signals are attatched via import
```

Comment:
I think this should be imported in the app config

https://docs.djangoproject.com/en/1.9/ref/applications/#django.apps.AppConfig.ready